### PR TITLE
Mikefab search border refocus

### PR DESCRIPTION
**Pull request by mikefab**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "express-password-reset": "latest",
     "express-session": "~1.0.0",
     "file-loader": "^0.8.5",
-    "jquery": "^2.2.1",
+    "jquery": "^2.2.2",
```
Comment (gatoatigrado) [inline package.json]: why was this necessary?

File: public/model/searched-admins.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+var P = require('pjs').P;
+
+var SearchedAdmins = P({
+  init: function(onUpdate) {
+    this.onUpdate = onUpdate;
+    // `searched_admin_codes` is a map from admin code to callbacks. The
+    // callbacks are called when the admin is unsearched.
+    this.searched_admin_codes = {};
+    // Temp hack to avoid double refocus on search select
```
Comment (gatoatigrado) [inline public/model/searched-admins.js]: Yeah, try to use the event emitter stuff ... this is kind of a perfect use case.

File: package.json (hunk 1 above)
Comment (mikefab) [inline package.json]: This line is not needed.

**Comment by mikefab** [comment]
I created a new branch off master and extended the selected-admins to include admin-search functionality. I will close this pull request and open a new one for the other branch: mikefab-search-admin.
